.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+    "tabWidth": 2,
+    "printWidth": 100,
+    "singleQuote": true,
+    "trailingComma": "es5",
+    "jsxSingleQuote": false,
+    "bracketSpacing": true,
+    "arrowParens": "always",
+    "embeddedLanguageFormatting": "auto",
+    "endOfLine": "lf",
+    "insertPragma": false,
+    "proseWrap": "preserve",
+    "quoteProps": "consistent",
+    "requirePragma": false
+}
```

Comment:
You should always include a newline at the end of your files (GitHub is marking it to show you it's missing).  The reason we do this is so that files don't get concatenated together, and the end of one file be on the same line as the start of another.